docs/integrations/sources/github.md

```diff
@@ -17,45 +17,44 @@ This connector outputs the following full refresh streams:
 * [Organizations](https://docs.github.com/en/rest/reference/orgs#get-an-organization)
 * [Pull request comment reactions](https://docs.github.com/en/rest/reference/reactions#list-reactions-for-a-pull-request-review-comment)
-* [Pull request stats](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request)
 * [Pull request commits](https://docs.github.com/en/rest/reference/pulls#list-commits-on-a-pull-request)
 * [Repositories](https://docs.github.com/en/rest/reference/repos#list-organization-repositories)
-* [Reviews](https://docs.github.com/en/rest/reference/pulls#list-reviews-for-a-pull-request)
 * [Tags](https://docs.github.com/en/rest/reference/repos#list-repository-tags)
 * [Teams](https://docs.github.com/en/rest/reference/teams#list-teams)
 * [Users](https://docs.github.com/en/rest/reference/orgs#list-organization-members)
 
 This connector outputs the following incremental streams:
 
 * [Comments](https://docs.github.com/en/rest/reference/issues#list-issue-comments-for-a-repository)
-* [Commits](https://docs.github.com/en/rest/reference/repos#list-commits)
 * [Commit comments](https://docs.github.com/en/rest/reference/repos#list-commit-comments-for-a-repository)
+* [Commits](https://docs.github.com/en/rest/reference/repos#list-commits)
+* [Deployments](https://docs.github.com/en/rest/reference/deployments#list-deployments)
 * [Events](https://docs.github.com/en/rest/reference/activity#list-repository-events)
-* [Issues](https://docs.github.com/en/rest/reference/issues#list-repository-issues)
 * [Issue events](https://docs.github.com/en/rest/reference/issues#list-issue-events-for-a-repository)
 * [Issue milestones](https://docs.github.com/en/rest/reference/issues#list-milestones)
+* [Issues](https://docs.github.com/en/rest/reference/issues#list-repository-issues)
+* [Project cards](https://docs.github.com/en/rest/reference/projects#list-project-cards)
+* [Project columns](https://docs.github.com/en/rest/reference/projects#list-project-columns)
 * [Projects](https://docs.github.com/en/rest/reference/projects#list-repository-projects)
+* [Pull request stats](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request)
 * [Pull requests](https://docs.github.com/en/rest/reference/pulls#list-pull-requests)
 * [Releases](https://docs.github.com/en/rest/reference/repos#list-releases)
 * [Review comments](https://docs.github.com/en/rest/reference/pulls#list-review-comments-in-a-repository)
+* [Reviews](https://docs.github.com/en/rest/reference/pulls#list-reviews-for-a-pull-request)
 * [Stargazers](https://docs.github.com/en/rest/reference/activity#list-stargazers)
-* [Deployments](https://docs.github.com/en/rest/reference/deployments#list-deployments)
-* [Project cards](https://docs.github.com/en/rest/reference/projects#list-project-cards)
-* [Project columns](https://docs.github.com/en/rest/reference/projects#list-project-columns)
 
 ### Notes
```

Comment:
Can we add a note here that explains why some streams are supported for start date and some are not?